servicetalk-opentracing-zipkin-publisher/src/main/java/io/servicetalk/opentracing/zipkin/publisher/reporter/UdpReporter.java

```diff
@@ -139,8 +138,7 @@ public Builder codec(Codec codec) {
         }
 
         /**
-         * Sets an {@link IoExecutor} to use for writing to the datagram channel. Defaults to the
-         * {@link GlobalExecutionContext} {@link IoExecutor}.
+         * Sets an {@link IoExecutor} to use for writing to the datagram channel.
```

Comment:
We usually do not clarify default values in javadoc. Removed it to avoid referencing internal class from the public javadoc.